### PR TITLE
Fixes for the Antares form

### PR DESCRIPTION
**Pull request by dsavchenko**
Resolves https://github.com/oda-hub/frontend-astrooda/issues/6 
Should resolve https://github.com/oda-hub/frontend-astrooda/issues/7

_(Validation not yet working. Suggestions?)_

### CODE REVIEW

File: instruments/astrooda_antares/astrooda_antares.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+jQuery(document).ready(function($){
```
Comment (volodymyrss) [inline instruments/astrooda_antares/astrooda_antares.js:1]: thanks cool! Need to give Mohamed to review, let's try to agree on this flow on Monday.

Comment (dsavchenko) [reply instruments/astrooda_antares/astrooda_antares.js:1]: This is just an attempt at copy-paste programming
The validation still doesn't work

**Comment by volodymyrss** [comment]
> This is just an attempt at copy-paste programming
The validation still doesn't work 

oh well. I am sure you can find the way to debug. Browser dev console very handy for this. Though if it's too complex, we can resort to MM.

**Comment by dsavchenko** [comment]
Now it works
I wanted to have the code in a separate file, but this way is susceptible to race conditions. Anyway, some instrument-specific things like scw_list validation are already there.

**Comment by volodymyrss** [comment]
Ok! Let's make @motame  review it anyway, lets discuss on Monday.

File: instruments/astrooda_antares/astrooda_antares.inc [hunk 2]
```diff
@@ -26,6 +26,7 @@ function astrooda_antares($form, &$form_state) {
 
   $astrooda_settings = variable_get('astrooda_settings');
   $instrument_defaults= $astrooda_settings['instruments']['antares'];
+
```
Comment (volodymyrss) [inline instruments/astrooda_antares/astrooda_antares.inc:29]: not good!

File: instruments/astrooda_antares/astrooda_antares.install [hunk 3]
```diff
@@ -38,13 +38,9 @@ function astrooda_antares_install() {
       'enabled' => TRUE,
       'name' => $instrument_info['name'],
       'title' => $instrument_info['title'],
-      'E1_keV' => 20.0,
-      'E2_keV' => 100.0,
-      'radius' => 15,
+      'radius' => 2,
```
Comment (volodymyrss) [inline instruments/astrooda_antares/astrooda_antares.install:41]: this is good! for antares

**Review by volodymyrss — APPROVED** [review]

**Review by motame — APPROVED** [review]
On would use the BS validation but its OK. See below table


Option | HTML attribute | Type | Description
-- | -- | -- | --
inclusive | data-bv-between-inclusive | Boolean | Can be true or false. If true, the input value must be in the range strictly
max* | data-bv-between-max or max | Float | The upper value in the range.It's a dynamic option
message | data-bv-between-message | String | The error message
min* | data-bv-between-min or min | Float | The lower value in the range.It's a dynamic option

**Comment by dsavchenko** [comment]
> One would use the Bootstrap validation

Thank you, I will try this option. 
It should be better, as this way validation related things move from general oda module to the Antares module.
I will include it here and request another review.

**Pull request merged**